package.json

```diff
@@ -6,24 +6,24 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^18.2.12",
-    "@angular/common": "^18.2.12",
-    "@angular/compiler": "^18.2.12",
-    "@angular/core": "^18.2.12",
-    "@angular/forms": "^18.2.12",
-    "@angular/platform-browser": "^18.2.12",
-    "@angular/platform-browser-dynamic": "^18.2.12",
-    "@angular/router": "^18.2.12",
+    "@angular/animations": "^19.1.3",
+    "@angular/common": "^19.1.3",
+    "@angular/compiler": "^19.1.3",
+    "@angular/core": "^19.1.3",
+    "@angular/forms": "^19.1.3",
+    "@angular/platform-browser": "^19.1.3",
+    "@angular/platform-browser-dynamic": "^19.1.3",
+    "@angular/router": "^19.1.3",
     "rxjs": "~6.6.0",
```

Comment:
Pls also update rxjs, @types/jnode, typescript to the same versions as in the dashboard.